layouts/shortcodes/self-assessment-latest.html

```
@@ -0,0 +1,2 @@
+<self-assessment></self-assessment>
+<script src="https://pkic.github.io/self-assessment/{{ with .Get "version" }}{{ . }}{{ else }}develop{{ end }}/self-assessment.js"></script>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)